templates/form.html

```diff
@@ -3,25 +3,52 @@
 <div class="row justify-content-center">
-    <div class="col-lg-8 col-sm-10 mb-3">
-        <div class="card">
-          <h4 class="card-header">{{ _("Server configuration") }}</h4>
-          <div class="alert alert-info card-header">
-              <i class="fa fa-info-circle" aria-hidden="true"></i>
-              {{ _("This is the domain that will be used to name and reach your internet cube. If you do not already own a domain name, you can pick a free domain ending with .nohost.me, .noho.st or .ynh.fr. It will automatically be configured during the setup.") }}
-          </div>
-          <div class="card-body">
-              <div class="form-group row">
-                  <label for="main_domain" class="col-3 col-form-label">{{ _("Main domain") }}</label>
-                  <div class="col-9">
-                      <input type="text" class="validate form-control" name="main_domain" id="main_domain" placeholder="{{ _("myserver.nohost.me") }}" pattern="^([^\W_A-Z]+([-]*[^\W_A-Z]+)*\.)+([^\W\d_]{2,})$">
-                      <div class="invalid-feedback default">
-                          {{ _("Please specify a valid domain") }}
-                      </div>
-                  </div>
-              </div>
-          </div>
-        </div>
+	<div class="col-lg-8 col-sm-10 mb-3">
+		<div class="card">
+			<h4 class="card-header">{{ _("Server configuration") }}</h4>
+			<div class="alert alert-info card-header">
+				<i class="fa fa-info-circle" aria-hidden="true"></i>
+				{{ _("This is the domain that will be used to name and reach your internet cube. If you do not already own a domain name, you can pick a free domain ending with .nohost.me, .noho.st or .ynh.fr. It will automatically be configured during the setup.") }}
+			</div>
+			<div class="card-body">
+				<div class="form-group row">
+					<label for="main_domain" class="col-3 col-form-label">{{ _("Main domain") }}</label>
+					<div class="col-9">
+						<input type="text" class="validate form-control" name="main_domain" id="main_domain" placeholder="{{ _("myserver.nohost.me") }}" pattern="^([^\W_A-Z]+([-]*[^\W_A-Z]+)*\.)+([^\W\d_]{2,})$">
+						<div class="invalid-feedback default">
+							{{ _("Please specify a valid domain") }}
+						</div>
+					</div>
+				</div>
+				<div class="subscribe-password-div">
+					<div class="form-group row">
+						<div class="alert alert-info card-header">
+							<i class="fa fa-info-circle" aria-hidden="true"></i>
+							{{ _("This kind of domain is freely provided by the YunoHost project. Please remember the password you enter, it will be used to keep control of the domain even if you loose control of your server.") }}
+						</div>
+						<label for="domain_password" class="col-3 col-form-label">{{ _("Domain password") }}</label>
+						<div class="col-9">
+							<input type="password" class="validate form-control" name="domain_password" id="domain_password" pattern="^.{8,}$">
```

Comment:
I recommend 12 chars at least in this case.

Reply:
Done in https://github.com/labriqueinternet/install/pull/7/commits/1d809218b7937e4d6681640aa251017c380b757b